hllplus/hllplus.go

```diff
@@ -82,14 +92,23 @@ func (s *HLL) Merge(other *HLL) {
 		return
 	}
 
+	// FIXME: allow sparse merge
```

Comment:
No need to fix this as part of this PR, we can do the sparse/sparse + sparse/dense merging separately later